### DISCUSSION TITLE
【灌水】在原神前瞻直播间当了半小时的兼职巡海游侠

### DISCUSSION THREAD

**Discussion by GanlinSama** [share121/inter-knot#2411] [其他]
本人中午无意间划入原神直播间，因看不惯黑子逆天发言于是临时起意兼职了巡海游侠，斗争半小时后见形势略有好转便离开战场（~~饭点到了~~）
不知其他同僚战绩如何<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by hua-yuanshuai** [comment]
举报了一堆3°和褒姒<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by Brighttaz** [comment]
不愧是巡海游侠，点了

**Comment by LUOFENGMENG** [comment]
真棒<img src="https://bbs-static.miyoushe.com/static/2023/07/12/041ab3fd1c16a616b703e0c254cdc4ad_7800210310515169099.png" alt="帕姆_点赞" width="75" >

**Comment by HKLHaoBin** [comment]
我设置了太多屏蔽词 导致我一点节奏都看不到

<img src="https://bbs-static.miyoushe.com/static/2024/08/07/86c603f895bd4e13b331508610a488fc_4608730387935880969.png" alt="流浪者-默认同意" width="75" >
